datalad/distribution/update.py

```diff
@@ -394,9 +394,7 @@ def _plain_merge(repo, _, target, merge_opts=None):
 
 def _annex_plain_merge(repo, _, target, merge_opts=None):
     yield from _plain_merge(repo, _, target, merge_opts=merge_opts)
-    # Note: Avoid repo.merge_annex() so we don't needlessly create synced/
-    # branches.
-    repo.call_git(["annex", "merge"])
+    repo.localsync()
```

Comment:
Hmm, I still prefer `annex merge` because all this is trying to do is trigger a merge of the `git-annex` branches.  (Which means I didn't leave a very good comment because it only gave a superficial reason for avoing the `git annex sync`-based `AnnexRepo.merge_annex()`.)

Reply:
OK, will revert.

Reply:
Done